0004-oauth/README.md

```diff
@@ -297,6 +297,13 @@ The following rules must be enforced by the AS when receiving a token request.
 - Ensure that the client authentication method is the same as the one used during PAR (if `client_assertion` was used during PAR, the same authentication method **must** be used, with a JWT containing a distinct `jti` claim, but signed by the same key)
 - DPoP proof and client assertion **must** be signed using a different keypair.
 - The Token Response from the AS **must** contain a `sub` claim, which must be the end-user's ATPROTO identifier (`did:plc:123`)
+- The validity of the access token **must** be limited to a short period of time (typically 1-5 minutes). The access token validity **must not** exceed 1 hour.
```

Comment:
I'd be a bit more generous and say "5 to 15 minutes".

@devinivy what are the current access token limits in prod with the legacy auth system?

Reply:
It seems to be either 15s (for "short" session) or 2min.

Reply:
I believe it's 2hr, not 2min.  Some of these defaults are informed by common defaults that are actually used, e.g. by services such as Auth0.